app/models/miq_server/server_smart_proxy.rb

```diff
@@ -82,6 +82,9 @@ def queue_call(ost)
         if target.kind_of?(ManageIQ::Providers::Openstack::CloudManager::Vm) ||
            target.kind_of?(ManageIQ::Providers::Openstack::CloudManager::Template)
           timeout_adj = 4
+        elsif target.kind_of?(ManageIQ::Providers::Azure::CloudManager::Vm) ||
+              target.kind_of?(ManageIQ::Providers::Azure::CloudManager::Template)
+          timeout_adj = 4
```

Comment:
In the future, we should probably have these classes implement a timeout function so we can just ask the target what it's timeout should be.

Reply:
Yup I said something to that effect in the description of the PR.